pyproject.toml

```
@@ -0,0 +1,64 @@
+[tool.poetry]
+name = "simple-settings"
+version = "1.1.0"
+description = "A simple way to manage your project settings."
+authors = ["Diego Garcia <drgarcia1986@gmail.com>"]
+license = "MIT"
+keywords=["django", "flask", "bottle", "tornado", "settings", "configuration", "conf"]
+classifiers=[
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 3',
+]
+packages = [
+    { include = "simple_settings" },
+    { include = "strategies", from = "simple_settings" },
+    { include = "dynamic_settings", from = "simple_settings" },
+]
+repository = "https://github.com/drgarcia1986/simple-settings"
+
+[tool.poetry.urls]
+"Download" = "https://github.com/drgarcia1986/simple-settings/tarball/master"
+
+[tool.poetry.dependencies]
+python=">=3.6.1,<4.0"
+jsonpickle = { version = "2.0.0", optional = true }
+toml = { version = "0.10.2", optional = true }
+PyYAML = { version = "6.0", optional = true }
+SQLAlchemy = { version = "1.4.26", optional = true }
+pymemcache = { version = "3.5.0", optional = true }
+six = { version = "1.16.0", optional = true}
+redis = { version = "3.5.3", optional = true }
+boto3 = { version = "1.9.201", optional = true }
+
+
+[tool.poetry.dev-dependencies]
+bumpversion="0.6.0"
+flake8="4.0.1"
+isort="5.9.3"
+moto="2.3.0"
+pytest-cov="3.0.0"
+pytest="6.2.5"
+wheel="0.37.1"
```

Same suggestion as above about loosening the version specification.

```suggestion
bumpversion="^0.6.0"
flake8="^4.0.1"
isort="^5.9.3"
moto="^2.3.0"
pytest-cov="^3.0.0"
pytest="^6.2.5"
wheel="^0.37.1"
```